.github/workflows/coverage.yml

```diff
@@ -21,7 +21,26 @@ jobs:
     - name: System dependencies
       run: |
         sudo apt-get update
-        sudo apt-get -o Acquire::Retries=3 install build-essential libgmp-dev z3 cvc4 opam cargo verilator
+        sudo apt-get -o Acquire::Retries=3 install build-essential libgmp-dev z3 cvc4 opam cargo verilator git curl
+
+    - name: Install Lean version manager
+      run: |
+        curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | bash -s -- -y
+
+    - name: Restore cached elan
+      id: cache-elan-restore
+      uses: actions/cache/restore@v3
+      with:
+        path: ~/.elan
+        key: ${{ matrix.os }}-${{ matrix.version }}-cov
```

Comment:
Can we use the same key here as our opam cache, or will they end up clobbering each other?

Reply:
I think they are going to clobber each other actually.

Reply:
(so I would prefix the key by `elan-`)